services/postgres/docker-entrypoint-initdb.d/create-readonly-user.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+set -e
+
+#
+# This script are ONLY run if you start the container with a data directory that is EMPTY (i.e. first time)
```

Comment:
I agree with @GitHK. It looks like simcore repository is not the right place to have this logic.

Reply:
@YuryHrytsuk @GitHK Ok, let me ask you a question: 
- where and how is the admin user created? 
- why this user should be created differently to the admin user?
- what happens when you change POSTGRES_*_PASSWORD?

Reply:
If this is going to be used for local deployment, it is absolutely fine. For master / staging / prod deployments, we may try to add some logic into OPS repositories.

Reply:
> * where and how is the admin user created?

The admin user is created:
* for deployments with self-hosted Postgres during first Postgres initialization by postgres container
* for AWS deployments: during RDS initialization (it is done via terraform and incorporated in terraform code)

The main point here: the user is created for both self-hosted and AWS-host Postgres

> * why this user should be created differently to the admin user?

I don't have a strong opinion here. It is fine if it is created the same way (not sure if it is possible). The problem with this PR is: it introduces new logic which does not apply for AWS Database.

> * what happens when you change POSTGRES_*_PASSWORD?

based on my quick research:
* for self-hosted postgres: it will not change user's password but it will break all running simcore services (they will use wrong password)
* AWS-hosted postgres will update password

Reply:
I see. For the moment, this PR removes the initialization script in postgres. Instead i created a template sql script to create a read-only user that depends on the repo.configs. 
Since we have two types of database deploys we can initialy set these up via adminer.